### PR TITLE
[What's Happening WIP] v2 announcement

### PR DESCRIPTION
**Pull request by RichKummer**
### Related Ticket(s)

#1564 

### Description

- New "What's Happening" page added 

### Changelog

**New**

- "What's Happening" page

**Changed**

- nav file

**Removed**

- N/A

### CODE REVIEW

File: src/data/nav-items.yaml [hunk 1]
```diff
@@ -37,6 +37,10 @@
       path: /contributing/components
     - title: Patterns
       path: /contributing/patterns
+
+- title: What's Happening
```
Comment (oliviaflory) [inline src/data/nav-items.yaml:41]:
```suggestion
- title: What’s Happening
```

File: src/pages/whats-happening/index.mdx [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
+title: What's happening
```
Comment (oliviaflory) [inline src/pages/whats-happening/index.mdx:2]:
```suggestion
title: What’s happening
```

**Comment by oliviaflory** [comment]
Thank you @RichKummer!! 

I shared the LTS image with @kennylam today, so we can link to that once he's had a chance to post it on our dotcom repo.

I will give this a full / better read on Monday, but adding a few small tweaks:

1. Can we move the `What's happening` page to the second spot in the left nav? ie below `About Carbon for IBM.com` and above `Designing`
2. Need to adjust to lower case `happening` in left nav
3. Left nav and lead space need to use the real apostrophe (per Mike). I left two inline comments that should fix it I think.
4. Forgot we need to find a place to link the recording and presentation. 
5. I'm wondering if we should hold on the What's changing section and update at a later date. Since we aren't listing every component that's changing and we don't have a list of the deprecated ones quite yet?
Screenshot 2023-06-02 at 5 07 02 PM

File: src/pages/whats-happening/index.mdx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+</ResourceCard>
+  </Column>
+
+</Row>
+
+### What are the benefits?
+
+- Fully support Carbon v11 updates to color and type tokens. Also includes Carbon v11 component and accessibility updates
+- Improvements to package build which make for faster compilation time
```
Comment (kennylam) [inline src/pages/whats-happening/index.mdx:50]:
```suggestion
- Faster package build times
```

File: src/pages/whats-happening/index.mdx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+- Faster page load time
+- Upgrade of Web Components Lit v1 to Lit v2
```
Comment (kennylam) [inline src/pages/whats-happening/index.mdx:52]:
```suggestion
- Web Components upgraded to use Lit v2
```

File: src/pages/whats-happening/index.mdx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+- Simplified component list with more flexibility for each component
+- Component enhancements for authoring such as content section and Masthead v2
+- Improvements to spacing logic across Layout and UI components
```
Comment (kennylam) [inline src/pages/whats-happening/index.mdx:55]:
```suggestion
- Improvements to spacing logic across layout and UI components
```

File: src/pages/whats-happening/index.mdx [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+Adopters should plan to migrate their application, webpages, and/or docs to Carbon for IBM.com v2 _before the end of maintenance of v1_. Ensure you have built the time for migration into your planning for 2024. If you remain on v1 after the end of maintenance, your application could have a risk with unresolved security vulnerabilities and/or outdated upstream dependencies. See Long Term Support (LTS) for the timeline above.
+
+#### What frameworks will be supported?
+
+With Carbon for IBM.com v2, the team will officially support the following frameworks:
+
+- Carbon for IBM.com Web Components
+- Carbon for IBM.com Web Components with React wrappers
+- Carbon Web Components v2
```
Comment (kennylam) [inline src/pages/whats-happening/index.mdx:86]:
```suggestion
Carbon for IBM.com v2 will no longer support React and focus fully on web components. This applies to the [@carbon/web-components](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/carbon-web-components) and [@carbon/ibmdotcom-web-components](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/web-components) packages. Both packages will still provide React wrappers for projects requiring React.

- Carbon for IBM.com Web Components
- Carbon for IBM.com Web Components with React wrappers
- Carbon Web Components v2
```

File: src/pages/whats-happening/index.mdx [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+#### Will Carbon for IBM.com React v1 be deprecated?
+
+Yes, all teams that rely on the Carbon for IBM.com React package can migrate to the React wrappers.
```
Comment (kennylam) [inline src/pages/whats-happening/index.mdx:90]: Not sure if this is still necessary with the changes to the previous question. But we can leave it in if you think it provides more clarity.

**Review by kennylam — COMMENTED** [review]
@RichKummer Looking good! I just added a few suggestions.

**Comment by kennylam** [comment]
> I shared the LTS image with @kennylam today, so we can link to that once he's had a chance to post it on our dotcom repo.

@oliviaflory Looks like Rich already added the new image to this documentation from the previous PR I made. I'll wait for this to publish live and use that image in the repo `release-schedule` page – it's easier to reference that way. Then moving forward, we can make any updates to the LTS image in the website and it will update everywhere else it's used.

**Review by kennylam — APPROVED** [review]
LGTM!

File: src/pages/whats-happening/index.mdx [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+Adopters should plan to migrate their application, webpages, and/or docs to Carbon for IBM.com v2 _before the end of maintenance of v1_. Ensure you have built the time for migration into your planning for 2024. If you remain on v1 after the end of maintenance, your application could have a risk with unresolved security vulnerabilities and/or outdated upstream dependencies. See Long Term Support (LTS) for the timeline above.
+
+#### What frameworks will be supported?
+
+Carbon for IBM.com v2 will no longer support React and focus fully on web components. This applies to the [@carbon/web-components](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/carbon-web-components) and [@carbon/ibmdotcom-web-components](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/web-components) packages. Both packages will still provide React wrappers for projects requiring React.
+
+- Carbon for IBM.com Web Components
+- Carbon for IBM.com Web Components with React wrappers
+- Carbon Web Components v2
```
Comment (kennylam) [inline src/pages/whats-happening/index.mdx:86]:
```suggestion

```

Whoops, I forgot to remove these from my previous suggestion.

**Comment by RichKummer** [comment]
> > I shared the LTS image with @kennylam today, so we can link to that once he's had a chance to post it on our dotcom repo.
> 
> @oliviaflory Looks like Rich already added the new image to this documentation from the previous PR I made. I'll wait for this to publish live and use that image in the repo `release-schedule` page – it's easier to reference that way. Then moving forward, we can make any updates to the LTS image in the website and it will update everywhere else it's used.

Hey @kennylam just a heads up, in my last commit I updated the image on the page itself to a link out to your PR update. This way it will be easier for you to add any table or explanation if you wanted (similar to [Carbon core's release](https://github.com/carbon-design-system/carbon/blob/main/docs/release-schedule.md))

**Comment by kennylam** [comment]
@RichKummer I saw that, thanks!

**Pull request merged**